apps/dashboard/package.json

```diff
@@ -14,6 +14,7 @@
     "@apollo/experimental-nextjs-app-support": "^0.4.3",
     "@emotion/react": "^11.11.1",
     "@emotion/styled": "^11.11.0",
+    "@galoymoney/client": "0.1.62",
```

Comment:
we don't want to use `@galoymoney/client` anymore. it was mostly used at a time when we didn't use the autogenerated function from our graphql schema.